provisioning/development-osx.yml

```diff
@@ -0,0 +1,7 @@
+---
+- hosts: localhost
+  # remote_user: ubuntu
+  # sudo: yes
```

Comment:
this is the new `become: true`, we need to understand if needed for more tasks or we should only use it for specific tasks